ui/details-panel/AnnotationsTab.jsx

```diff
@@ -0,0 +1,235 @@
+      } else {
+        this.thNotify.send("No classification on this job to delete", 'warning');
+      }
+    });
+  }
+
+  deleteClassification(classification) {
+    const jobMap = this.ThResultSetStore.getJobMap();
+    const job = jobMap[`${classification.job_id}`].job_obj;
+
+    job.failure_classification_id = 1;
+    this.ThResultSetStore.updateUnclassifiedFailureMap(job);
+
+    classification.delete().then(
+      () => {
+        this.thNotify.send("Classification successfully deleted", "success");
+        // also be sure the job object in question gets updated to the latest
+        // classification state (in case one was added or removed).
+        this.ThResultSetStore.fetchJobs([job.id]);
+        this.$rootScope.$emit(
+          this.thEvents.jobsClassified,
+          { jobs: { [job.id]: job } }
+        );
+      },
+      () => {
+        this.thNotify.send(
+          "Classification deletion failed",
+          "danger",
+          { sticky: true }
+        );
+      });
+  }
+
+  deleteBug(bug) {
+    const { selectedJob } = this.props;
+
+    bug.delete()
+      .then(() => {
+          this.thNotify.send(
+            `Association to bug ${bug.bug_id} successfully deleted`,
+            "success"
+          );
+          this.$rootScope.$emit(
+            this.thEvents.bugsAssociated,
+            { jobs: { [selectedJob.id]: selectedJob } }
+          );
+        }, () => {
+          this.thNotify.send(
+            `Association to bug ${bug.bug_id} deletion failed`,
+            "danger",
+            { sticky: true }
+          );
+        }
+      );
+  }
+
+
+  render() {
+    const {
+      $injector, classifications, classificationTypes,
+      bugs, getBugUrl
+    } = this.props;
+    const dateFilter = $injector.get('$filter')('date');
+
+    return (
+      <div className="row h-100">
+        <div className="col-sm-10 classifications-pane job-tabs-content">
+          {classifications && classifications.length > 0 ?
+            <AnnotationsTable
+              classifications={classifications}
+              dateFilter={dateFilter}
+              deleteClassification={this.deleteClassification}
+              classificationTypes={classificationTypes}
+            /> :
+            <p>This job has not been classified</p>
+          }
+        </div>
+
+        {classifications && classifications.length > 0 && bugs &&
+        <div className="col-sm-2 bug-list-pane">
+          <RelatedBug
+            bugs={bugs}
+            getBugUrl={getBugUrl}
+            deleteBug={this.deleteBug}
+          />
+        </div>}
+      </div>
+    );
+  }
+}
+
+AnnotationsTab.propTypes = {
+  classifications: PropTypes.array,
+  $injector: PropTypes.object,
+  classificationTypes: PropTypes.object,
+  getBugUrl: PropTypes.func,
+  bugs: PropTypes.array,
+  selectedJob: PropTypes.object,
+};
+
+treeherder.directive('annotationsPanel', ['reactDirective', '$injector', (reactDirective, $injector) =>
```

Comment:
Do you think the directive name be changed to `annotationsTab` too, or should we save that until later?

Reply:
Ahh, good call. I'll do that prior to merging.  Thanks for the quick review!